src/rqt_graph/dotcode.py

```diff
@@ -498,41 +654,98 @@ def generate_dotgraph(self,
                                              rankdir=orientation)
 
         ACTION_TOPICS_SUFFIX = '/action_topics'
-        namespace_clusters = {}
-        for n in (nt_nodes or []) + [action_prefix + ACTION_TOPICS_SUFFIX for (action_prefix, _) in action_nodes.items()]:
+        IMAGE_TOPICS_SUFFIX = '/image_topics'
+
+        namespace_clusters = self._populate_node_graph(cluster_namespaces_level, (nt_nodes or [])
+                                    + [action_prefix + ACTION_TOPICS_SUFFIX for (action_prefix, _) in action_nodes.items()]
+                                    + [image_prefix + IMAGE_TOPICS_SUFFIX for (image_prefix, _) in image_nodes.items()]
+                                    + nn_nodes if nn_nodes is not None else [],
```

Comment:
Please wrap after the open parenthesis and make the operators trailing the previous lines.